apollo-router/tests/integration/validation.rs

```diff
@@ -128,7 +128,7 @@ async fn test_validation_error() {
           }
         },
         {
-          "message": "Field \"topProducts\" of type \"Product\" must have a selection of subfields. Did you mean \"topProducts { ... }\"?",
+          "message": "Field \"topProducts\" of type \"Query\" must have a selection of subfields. Did you mean \"topProducts { ... }\"?",
```

Comment:
The previous message was wrong. https://github.com/apollographql/apollo-rs/pull/865 fixed it.